### PR TITLE
Ian

### PR DESCRIPTION
**Pull request by ianpokorny**
added util foler, helper.py, started SPI_Test

### CODE REVIEW

File: IanProjects/SPI_TEST/SPI_TEST.cpp
Comment (SnazzyBlanket) [inline IanProjects/SPI_TEST/SPI_TEST.cpp:1]: TAKE A HIKE

Comment (SnazzyBlanket) [inline IanProjects/SPI_TEST/SPI_TEST.cpp:1]: i dont like your nonsense

why all the commented out code?

be more clear

Comment (ianpokorny) [reply IanProjects/SPI_TEST/SPI_TEST.cpp:1]: get bent

**Review by SnazzyBlanket — APPROVED** [review]
eat a donkey

**Pull request merged**